client/my-sites/checkout/composite-checkout/wpcom/components/wp-checkout-order-summary.js

```diff
@@ -145,6 +146,42 @@ function CheckoutSummaryFeaturesListDomainItem( { domain } ) {
 	);
 }
 
+function CheckoutSummaryPlanFeatures() {
+	const translate = useTranslate();
+	const hasDomainsInCart = useHasDomainsInCart();
+	const planInCart = usePlanInCart();
+	const planFeatures = getPlanFeatures( planInCart, translate, hasDomainsInCart );
+
+	return (
+		<>
+			{ planFeatures &&
+				planFeatures.map( ( feature ) => {
+					return (
+						<CheckoutSummaryFeaturesListItem>
+							<WPCheckoutCheckIcon />
+							{ feature }
+						</CheckoutSummaryFeaturesListItem>
+					);
+				} ) }
+		</>
+	);
+}
+
+function getPlanFeatures( plan, translate, hasDomainsInCart ) {
+	if (
+		'business-bundle' === plan.wpcom_meta?.product_slug ||
+		'business-bundle-2y' === plan.wpcom_meta?.product_slug
+	) {
+		return [
+			! hasDomainsInCart && translate( 'Free domain for one year' ),
+			translate( 'Install plugins and themes' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'Install custom plugins and themes' )` *ES Score: 7*